gen.go

```diff
@@ -70,6 +70,10 @@ func (s *Schema) Generate() ([]byte, error) {
 			Definition: schema,
 		}
 
+		if !s.CheckForDuplicateTitles() {
+			panic(fmt.Errorf("duplicate titles detected for %s", context.Name))
```

Comment:
Would `return nil, fmt.Errorf(...)` work here? I don't imagine the goroutine backtrace would be helpful, so if the error propagates up in a nicer way that seems preferable.